src/RequestWrapper.php

```diff
@@ -119,35 +99,24 @@ class RequestWrapper
      *     @type int $retries Number of retries for a failed request.
      *           **Defaults to** `3`.
      *     @type array $scopes Scopes to be used for the request.
-     *     @type boolean $shouldSignRequest Whether to enable request signing.
+     *     @type bool $shouldSignRequest Whether to enable request signing.
      * }
      */
     public function __construct(array $config = [])
```

Comment:
Perhaps you should direct people to `RequestWrapperTrait::setCommonDefaults()` for shared option descriptions and keep just the wrapper-specific ones here? Precedent would be the change you made to the docs for `ServiceBuilder::pubsub()`.